app/assets/javascripts/common/reports.js

```diff
@@ -696,33 +708,14 @@ Reports = function ({
     const sumValues = monthOneValue + monthTwoValue + monthThreeValue;
     const threeMonthAverage = sumValues === 0 ? 0 : sumValues / 3;
 
-    const monthThreeIndexOfYear = monthIndexFromDateString(monthThreeDateKey);
+    const monthThreeIndexOfYear =
+      monthIndexFromChartLabelDateString(monthThreeDateKey);
     return {
       threeMonthAverage,
       monthThreeIndexOfYear,
     };
   }
 
-  function monthIndexFromDateString(dateString) {
-    const [month, year] = dateString.split("-");
-    const months = [
-      "jan",
-      "feb",
-      "mar",
-      "apr",
-      "may",
-      "jun",
-      "jul",
-      "aug",
-      "sep",
-      "oct",
-      "nov",
-      "dec",
-    ];
-
-    return months.indexOf(month.toLowerCase());
```

Comment:
Let's use Javascript built-in function to get the month number from month string.
```
new Date("jan-2023").getMonth()
```